routers/api/v1/utils/hook.go

```diff
@@ -69,7 +69,7 @@ func AddOrgHook(ctx *context.APIContext, form *api.CreateHookOption) {
 	org := ctx.Org.Organization
 	hook, ok := addHook(ctx, form, org.ID, 0)
 	if ok {
-		ctx.JSON(200, convert.ToHook(org.HomeLink(), hook))
+		ctx.JSON(201, convert.ToHook(org.HomeLink(), hook))
```

Comment:
maybe change to `http.StatusCreated` for readable.

Reply:
done

Reply:
done?

Reply:
Oh, the changes were lost when I rebased. I've readded the lost changes